docs/design/exclusive-volume.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Exclusive Volumes
+authors:
+  - "@hzxuzhonghu"
+reviewers:
+  - "@k82cn"
+  - "@zrss"
+approvers:
+  - "@k82cn"
+creation-date: 2019-11-21
+last-updated: 2019-11-21
+status: review
+---
+
+# Exclusive Volumes
+
+## Summary
+
+The purpose of this is to allow mount different volumes per pod of a Volcano Job.
+
+## Motivation
+
+Volume mount was supported from begin. But there are few limitations:
+
+1. We can specify volumes of Job scope, all the volumes are shared by all pods within a Job.
+
+2. We specify volumes by setting `TaskSpec.PodTemplateSpec.Volumes`, but similarly they are shared by pods within a task.
+
+But in real world, scenarios like DL/BigData, etc requires high performance. Shared storage has some performance issue, 
```

Comment:
I think that's not the motivation of this proposal :)

Reply:
I am not sure what others ?

Reply:
please talk with zrss for more detail about that.